framework/test/driver/BundleManifestTest.cpp

```diff
@@ -59,6 +59,14 @@ void TestUnicodeProperty(BundleContext bc)
 #endif
 }
 
+void TestDeepManifest(BundleContext bc)
+{
+  auto bundle = testing::InstallLib(bc, "TestBundleWithDeepManifest");
+  auto headers = bundle.GetHeaders();
+  US_TEST_CONDITION(headers.at(Constants::BUNDLE_SYMBOLICNAME).ToString() ==
```

Comment:
why check the symbolic name? whats the purpose of this test?

Reply:
The purpose of this test is to see `InstallLib` call finishes quickly. Based on our offline discussions, I'll change the test name to illustrate what it's doing